core/workspace_svg.js

```diff
@@ -340,29 +340,6 @@ Blockly.WorkspaceSvg.prototype.getOriginOffsetInPixels = function() {
 };
 
 /**
- * Return the injection div that is a parent of this workspace.
```

Comment:
Woops that is a mistake. Damn now `rerere` will forever mess that one up for me :(